packages/vue/src/composables/useAuth.ts

```diff
@@ -73,12 +76,12 @@ type UseAuth = () => ToComputedRefs<UseAuthReturn>;
 export const useAuth: UseAuth = () => {
   const { clerk, authCtx } = useClerkContext();
 
+  const getToken: GetToken = createGetToken(clerk);
+  const signOut: SignOut = createSignOut(clerk);
```

Comment:
Moved this out from the `computed` property since the value doesn't need to be recalculated